microprofile/WFLY-19588_microprofile_platform_7.adoc

```diff
@@ -0,0 +1,201 @@
+---
+categories:
+- microprofile
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/623
+feature-team:
+ developer: kabir
+ sme:
+  -
+ outside-perspective:
+  -
+---
+= [default] MicroProfile Platform 7.0
+:author:            Kabir Khan
+:email:             kkhan@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+This RFE is about updating WildFLy to support MicroProfile Platform 7.0. We currently support MicroProfile Platform 6.1.
+////
+
+////
+
+
+The major.minor versions of the individual specs are shown below. Micro updates are strictly for TCK fixes, and we should use the latest for the TCKs as these versions become available.
+
+Aside from the below information, both the 6.1 and 7.0 MicroProfile Platform version include the Jakarta EE 10 Core Profile.
+
+[cols="1,1,1,1,1"]
+|===
+|Spec | MP 6.1 Spec Version | Current Spec WildFly Version | MP 7.0 Spec Version | Updated Functionality (y/n)
+
+|Config
+|3.1
+|3.1
+|3.1
+|n
+
+|Fault Tolerance
+|4.0
+|4.0
+|4.1
+|y https://issues.redhat.com/browse/WFLY-19592[WFLY-19592]
+
+|Health
+|4.0
+|4.0
+|4.0
+|n
+
+|JWT
+|2.1
+|2.1
+|2.1
+|n
+
+|OpenAPI
+|3.1
+|3.1
+|4.0
+|y https://issues.redhat.com/browse/WFLY-19591[WFLY-19591]
+
+|Rest Client
+|3.0
+|3.0
+|4.0
+|y https://issues.redhat.com/browse/WFLY-19589[WFLY-19589]
+
+|Telemetry +
+|1.1
+|1.1
+|2.0
+|y https://issues.redhat.com/browse/WFLY-19589[WFLY-19589]
+
+|===
+
+There is no additional work in this RFE beyond what is done by the individual spec update RFEs, i.e. it is just an umbrella issue.
```

Comment:
I do agree, this is an umbrella issue, then I wonder how it is not being set as an epic, and include child issues, since the platform upgrade can't be certified _unless_ all of the child issues are resolved.

Reply:
AFAIK only Bilge creates Epics in our process

Reply:
Ok, thanks. Feel free to resolve.